src/Attribute/PropertyAttribute.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AutoMapper\Attribute;
+
+use AutoMapper\Extractor\PropertyMapping;
+
+interface PropertyAttribute
```

Comment:
not sure about this name... but it would be nice to share an interface between all attributes we're exposing